src/components/app.tsx

```diff
@@ -67,9 +70,9 @@ function App() {
   const mapCellsFromValues = (values: IValues) => {
     return (
       <>
-        {(Object.values(values)).map((val, i) => {
+        {(Object.values(values)).map((val) => {
           if (typeof val === "string" || typeof val === "number") {
-              return (<td key={i}>{val}</td>);
+              return (<td key={val}>{val}</td>);
```

Comment:
I wonder if React will get confused if you have two table cells with the same value.  Maybe you could use a composed key of the value and index like:

```
key={`${val}:${i}`}
```